src/Microsoft.ML.Data/Transforms/TransformBase.cs

```diff
@@ -244,7 +247,8 @@ public override bool IsColumnActive(int col)
     /// on multiple input columns.
     /// This class provides the implementation of ISchema and IRowCursor.
     /// </summary>
-    public abstract class OneToOneTransformBase : RowToRowMapperTransformBase, ITransposeDataView, ITransformCanSavePfa,
+    [BestFriend]
+    internal abstract class OneToOneTransformBase : RowToRowMapperTransformBase, ITransposeDataView, ITransformCanSavePfa,
         ITransformCanSaveOnnx
```

Comment:
There are a few `[BestFriend]` in this class, if you are doing another iteration, maybe we could make those public and remove the attribute. Not a requirement though.